universal-login-wallet/src/services/Services.ts

```diff
@@ -0,0 +1,17 @@
+const UniversalLoginSDK = require('universal-login-sdk').default;
+
+interface Config {
+  relayerUrl: string;
+  jsonRpcUrl: string;
+}
+
+export interface Services {
+  sdk: any;
+}
+
+const Services = (config: Config) => {
```

Comment:
This function being uppercase is misleading. If I were using it I would assume it's a class. You can change it to `createServices` **unless** it is the same in other projects - consistency is more important.